### ISSUE TITLE
anulus versus annulus - use of alternative forms in MP terms

### ISSUE THREAD

**Issue by anna-anagnostop** [mgijax/mammalian-phenotype-ontology#3847]
The following MP terms use the "anulus" variant in their primary term label:

[abnormal aortic valve anulus morphology](https://www.informatics.jax.org/vocab/mp_ontology/MP:0010598)
[abnormal pulmonary valve anulus morphology](https://www.informatics.jax.org/vocab/mp_ontology/MP:0010606)
[abnormal mitral valve anulus morphology](https://www.informatics.jax.org/vocab/mp_ontology/MP:0010613)
[abnormal tricuspid valve anulus morphology](https://www.informatics.jax.org/vocab/mp_ontology/MP:0010621)

In contrast, the following terms use the "annulus" variant:

[abnormal sperm annulus morphology](https://www.informatics.jax.org/vocab/mp_ontology/MP:0009834)
[absent sperm annulus](https://www.informatics.jax.org/vocab/mp_ontology/MP:0009835)
[abnormal annulus fibrosus morphology](https://www.informatics.jax.org/vocab/mp_ontology/MP:0021135)

I suggest that we use "annulus" (more prevalent form) in all above terms and add "anulus" as a synonym for consistency with the HPO.

**Comment by anna-anagnostop** [comment]
All relevant MP terms now use "annulus" in their primary label, with "anulus" added as an exact synonym.
